feature/floor-map/src/main/java/io/github/droidkaigi/confsched2023/floormap/FloorMapScreen.kt

```diff
@@ -69,26 +85,69 @@ private fun FloorMapScreen(
     uiState: FloorMapScreenUiState,
     snackbarHostState: SnackbarHostState,
     onSideEventClick: (SideEvent) -> Unit,
+    updateFloorLevelState: (FloorLevel) -> Unit,
 ) {
     Scaffold(
         modifier = Modifier.testTag(FloorMapScreenTestTag),
         snackbarHost = { SnackbarHost(snackbarHostState) },
         content = { padding ->
-            Column(
+            Box(
                 Modifier
-                    .padding(padding),
+                    .fillMaxSize()
+//                    .padding(padding) FIXME コメントを外すとSnackbarの分だけ下部のpaddingが合わなくなる,スナックバー表示中のUIもいただきたい
```

Comment:
I don't think we should use the snack bar for the floor map. Do you have any concerns other than the Snackbar?

Reply:
And sorry for interrupting, but can you use English as we are having other language speakers in our contributors? You can use DeepL or ChatGPT 🙇

Reply:
The remaining FIXME comments are in this file.


https://github.com/DroidKaigi/conference-app-2023/blob/e2d82275f0dea363f6b838c5fb2b5dcd4d6e186e/feature/floor-map/src/main/java/io/github/droidkaigi/confsched2023/floormap/component/FloorLevelSwitcher.kt